cime_config/config_grids.xml

```diff
@@ -4823,6 +4859,31 @@
       <map name="GLC2OCN_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_mpas.gis20km_to_oEC60to30v3_aave.181115.nc</map>
     </gridmap>
 
+    <gridmap glc_grid="mpas.gis20km" lnd_grid="ne30np4.pg2">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_gis20km_mono.230510.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_gis20km_bilin.230510.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_ne30pg2_mono.230510.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_ne30pg2_mono.230510.nc</map>
+    </gridmap>
```

Comment:
Are these actually needed?  I don't see a new `model_grid` specification that uses this combination of component grids.

Reply:
I don't entirely remember, but I think that we (Jon and I) decided to keep support (for now) for related bi-grid configurations. E.g., this could be used with the majority of this grid spec ...

```
    <model_grid alias="ne30pg2_r05_EC30to60E2r2_gis20" compset="_MALI">
      <grid name="atm">ne30np4.pg2</grid>
      <grid name="lnd">r05</grid>
      <grid name="ocnice">EC30to60E2r2</grid>
      <grid name="rof">r05</grid>
      <grid name="glc">mpas.gis20km</grid>
      <grid name="wav">null</grid>
      <mask>EC30to60E2r2</mask>
    </model_grid>
```
... to add support for a similar bi-grid configuration.

Reply:
@jonbob -- if bi-grids really are a thing of the past then I'm happy to remove support for these configs (I will also check w/ Charlie and Chloe to make sure that's ok with them).

Reply:
Chloe and Charlie are apparently ok w/ removing bi-grid support (they have been using tri-grid configs for their work). So if the consensus is to remove bi-grid support, I will put that on my to-do list as well.

Reply:
... and, based on a discussion w/ Jon yesterday, I'm going to table (resolve) this for now. We decided to leave in for now and tackle trimming down the support to only tri-grids in a future PR.